lib/fluent/command/templates/new_gem/test/plugin/test_filter.rb.erb

```diff
@@ -0,0 +1,16 @@
+require "helper"
+require "fluent/plugin/<%= plugin_filename %>"
+
+class <%= class_name %>Test < Test::Unit::TestCase
+  def setup
+    Fluent::Test.setup
+  end
+
+  def create_driver(conf)
+    Fluent::Test::Driver::Filter.new(Fluent::Plugin::<%= class_name %>).configure(conf)
+  end
+
+  def test_failure
```

Comment:
Use `test 'failure'` type testcases to introduce that feature to plugin authors.